source/api/extension/navplace/index.md

```diff
@@ -0,0 +1,331 @@
+---
+title: navPlace Extension
+layout: spec
+tags: [extension, navPlace]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [semantic versioning][notes-versioning].
+Changes will be tracked within the document.
+
+{% include copyright.md %}
+
+## Table of Contents
+{:.no_toc}
+
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Introduction
+
+The concept of place is foundational to human communication and understanding. Locations are quantitative descriptions of the position of a place, independent of time and other human or physical characteristics of such place. The concept of location is a first class descriptor for resources, and so needs a unique property. In IIIF Presentation API 3 there is no property designed for a geographic location.
+
+
+### 1.1 Objectives and Scope
+
+This IIIF Presentation API 3 extension describes a new property, `navPlace`, which is defined by earthbound geographic coordinates in the form of [GeoJSON-LD](https://geojson.org/geojson-ld/). Clients may use this property to leverage the functionality of web-based map platforms such as Google Earth, Leaflet, OpenLayers, etc., providing a means to enrich data presentation through map-based interfaces.
+
+Spatial coordinates for resources on other celestial bodies or contrived worlds can be expressed using the semantic pattern of GeoJSON. However, `navPlace` adopts the [existing GeoJSON specification](https://datatracker.ietf.org/doc/html/rfc7946) to promote interoperability with industry standard mapping libraries and methods using [WGS84](http://www.w3.org/2003/01/geo/wgs84_pos) as the coordinate reference system for projections of the surface of Earth. As such, expressing the location of extraterrestrial entities is not supported by the `navPlace` property. This extension does not preclude the development of future extensions to address this use case.
+
+
+### 1.2 Motivating Use Cases
+
+The reasons for associating geographic coordinates with web resources vary greatly. This extension does not intend to meet the data requirements for all geospatial practices. Use cases within the scope include
+
+
+*   Link a IIIF resource to a single or multiple geographic areas
```

Comment:
The bullets would read more smoothly if "Linking...', 'Supplying...', 'Representing...'